SPM/job_spm_single_results_display.m

```diff
@@ -0,0 +1,85 @@
+function job_spm_single_results_display(fspm, Ci, t1, Coordlist, output_dir, spmfile, wd)
+%%  INPUTS
+%   FSPM : cell variable defining the path to the SPM.mat file
+%   CI : integer variable with the number of the contrast to display
```

Comment:
maybe it's possible to give the "name" of the contrast, I must be possible to compare with the content of SPM.mat file